lib/utils/parser.rb

```diff
@@ -67,7 +67,19 @@ module LinuxMountParser
   # this parses the output of mount command (only tested on linux)
   # this method expects only one line of the mount output
   def parse_mount_options(mount_line, compatibility = false)
-    mount = mount_line.scan(/\S+/)
+    if includes_whitespaces? mount_line
+      # Device-/Sharenames and Mountpoints including whitespaces require special treatment:
+      # We use the keyword ' type ' to split up and rebuild the desired array of fields
+      type_split = mount_line.split(' type ')
```

Comment:
You might be able to use `rindex` to get the right-most ` type ` to avoid picking up a share name with the word ` type ` in it with spaces:

```ruby
type_index = mount_line.rindex(' type ')
fs_path = mount_line[0...type_index]
other_opts_index = type_index + 6 # since ' type ' is 6 chars
other_opts = mount_line[other_opts_index..-1]
# ... etc ...
```

I think this might decently well, but would need to be tested. Worth playing around with a bit?

Reply:
Definitely worth, but I left it out for now due to high complexity. Maybe later, but I cannot promise due to endofyear-crunch.

If one would address this edge-case, it might be clever to do the same for `BsdMountParser`, right? What would be your suggestion to keep it DRY?

Reply:
I'm OK leaving it out for this PR and we can circle back to it if someone raises a bug or if you feel like coming back for a second iteration. :)

I would definitely DRY it up if possible as long as it doesn't make it too complicated. I'm happy to help as needed!